modules/install-vault/install-vault

```diff
@@ -108,6 +115,14 @@ function user_exists {
   id "$username" >/dev/null 2>&1
 }
 
+function replace_text {
+  local readonly original_text_regex="$1"
+  local readonly replacement_text="$2"
+  local readonly string="$3"
+
+  sudo echo "$string" | sed "s|$original_text_regex|$replacement_text|g"
```

Comment:
Don't need `sudo` for this one

Reply:
Ha, is `sudo echo` ever necessary? :) Fixed.